From a60e812d6a364bf5198f35d4403a7996b1c85582 Mon Sep 17 00:00:00 2001
From: Chih-Yi Leu <cleu@mozilla.com>
Date: Thu, 22 Jun 2017 14:46:14 +0800
Subject: Add include guard to prevent duplicate inclusion


diff --git a/src/libANGLE/renderer/d3d/d3d11/dxgi_support_table.h b/src/libANGLE/renderer/d3d/d3d11/dxgi_support_table.h
index 1d8d685..1b31d01 100644
--- a/src/libANGLE/renderer/d3d/d3d11/dxgi_support_table.h
+++ b/src/libANGLE/renderer/d3d/d3d11/dxgi_support_table.h
@@ -8,6 +8,9 @@
 //   version, D3D feature level, and is sometimes guaranteed or optional.
 //
 
+#ifndef DXGI_SUPPORT_TABLE_H
+#define DXGI_SUPPORT_TABLE_H
+
 #include "common/platform.h"
 
 namespace rx
@@ -42,3 +45,5 @@ const DXGISupport &GetDXGISupport(DXGI_FORMAT dxgiFormat, D3D_FEATURE_LEVEL feat
 } // namespace d3d11
 
 } // namespace rx
+
+#endif
-- 
2.10.0.windows.1

