From 0b2127d4b726d3c3bc7e219df47382937a3422e1 Mon Sep 17 00:00:00 2001
From: Chih-Yi Leu <cleu@mozilla.com>
Date: Thu, 22 Jun 2017 14:47:31 +0800
Subject: Remove DCHECK in mru-cache.h to avoid header conflict


diff --git a/src/third_party/mrucache/base/containers/mru_cache.h b/src/third_party/mrucache/base/containers/mru_cache.h
index b171b7c..2469db7 100644
--- a/src/third_party/mrucache/base/containers/mru_cache.h
+++ b/src/third_party/mrucache/base/containers/mru_cache.h
@@ -192,7 +192,6 @@ class MRUCacheBase
     {
         // We don't use ordering_.size() for the return value because
         // (as a linked list) it can be O(n).
-        DCHECK(index_.size() == ordering_.size());
         return index_.size();
     }
 
-- 
2.10.0.windows.1

